docusaurus.config.js

```diff
@@ -271,7 +271,7 @@ module.exports = {
         docs: {
           sidebarPath: require.resolve("./sidebars.js"),
           routeBasePath: "docs",
-          exclude: ["**/shared/**", "**/app-dev-context", "**/concept-context"], // do not render "shared" content
+          exclude: ["**/app-dev-context/**", "**/concept-context/**"], // do not render "shared" content
```

Comment:
```suggestion
          exclude: ["**/app-dev-context/**", "**/concept-context/**"], // do not render context content
```
Sumpthin' like this?